src/libs/createRestaurantPK.js

```diff
@@ -0,0 +1,3 @@
+export const createRestaurantPK = (name, street, postalCode) => {
+    return name + street.replace(/ +/g, '') + postalCode;
```

Comment:
this probably needs to be a hash function rather than just combining the string together into one big string.
have a look at this - https://gist.github.com/kitek/1579117

Reply:
Oh!! Thanks!!